src/chrome/content/rules/KickassTorrents.xml

```diff
@@ -2,25 +2,29 @@
 	Requested: https://trac.torproject.org/projects/tor/ticket/9232
 	Updated: https://trac.torproject.org/projects/tor/ticket/10232
 
+	Since the above, updated from kickass.to to kickass.so
 -->
 <ruleset name="KickassTorrents">
   <target host="kat.ph" />
   <target host="*.kat.ph" />
+  <target host="kickass.so" />
+  <target host="www.kickass.so" />
   <target host="kickass.to" />
   <target host="www.kickass.to" />
   <target host="kickasstorrents.com" />
   <target host="www.kickasstorrents.com" />
 
+  <securecookie host="^kickass\.so$" name=".+" />
   <securecookie host="^kickass\.to$" name=".+" />
+  <securecookie host="^\.kat\.ph$" name=".*"/>
```

Comment:
It looks like the cookies are never set under the kat.ph or kickass.to domains anymore, even if you access the site from those addresses. I think these two lines can be removed since they are now always set under kickass.so.